packages/kbn-unified-data-table/src/components/compare_documents/hooks/use_comparison_cell_value.tsx

```diff
@@ -128,7 +178,12 @@ const FieldCellValue = ({ field, fieldName }: FieldCellValueProps) => {
           className={FIELD_NAME_CLASS}
           data-test-subj="unifiedDataTableComparisonFieldName"
         >
-          {field?.displayName ?? fieldName}
+          <>
+            {field?.displayName ?? fieldName}
```

Comment:
```suggestion
            {field?.displayName ?? fieldName}{' '}
```
Could we add a space between the field name and the icon here and in field statistics? Currently they look a bit crammed together, and I think some space would help:
<img width="694" alt="icon_space" src="https://github.com/elastic/kibana/assets/25592674/7a6d03b4-0f40-4349-90d9-f0fcaeedf184">